docs/pages/connect-your-client/tsh.mdx

```diff
@@ -1126,6 +1127,139 @@ $ tsh ssh node1.leaf1
 $ ssh node1.leaf1
 ```
 
+## Request access to resources
```

Comment:
This feels more like it belongs in an "access requests" section of the docs, not in a tsh section of the docs.